code/src/components/Task.js

```diff
@@ -0,0 +1,263 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+import tasks from 'reducers/todos/taskSlice';
+import moment from 'moment';
+import Check from '../assets/Check.svg';
+import Trash from '../assets/Trash.svg';
+import Lid from '../assets/Lid.svg';
+
+const Task = ({ taskData }) => {
+  const dispatch = useDispatch();
+  const onIsCompletedToggle = () => {
+    dispatch(tasks.actions.toggleItem(taskData.id));
+  };
+  const onRemoveTask = () => {
+    dispatch(tasks.actions.removeTask(taskData.id));
+  };
+  return (
+    <TaskWrapper>
+      <div className="task-row">
+        <CheckButton
+          role="button"
+          onClick={() => onIsCompletedToggle(taskData.id)}
+          complete={taskData.isCompleted}
+          onKeyDown={(e) => e.key === 'Enter' && onIsCompletedToggle(taskData.id)}
+          aria-label={
+            taskData.isCompleted
+              ? 'task complete. press enter to toggle.'
+              : 'task incomplete. press enter to toggle.'
+          }>
+          {taskData.isCompleted ? '' : <IncompleteCheckIcon src={Check} />}
+          {taskData.isCompleted ? <CheckIcon src={Check} /> : ''}
+        </CheckButton>
+
+        <p className="task-text">{taskData.text}</p>
+        <RemoveButton
+          role="button"
+          onClick={() => onRemoveTask(taskData.id)}
+          aria-label="Remove task">
+          <span className="trash">
+            <span src={Trash} />
+            <i src={Lid} />
+          </span>
+        </RemoveButton>
+      </div>
+      <Date>
+        Created {moment(taskData.date).format('ddd, MMM Do YYYY @ hh:mm a')}
+      </Date>
+    </TaskWrapper>
+  );
+};
+export default Task;
+
+const TaskWrapper = styled.div`
+  // background-color: #f2f7f8;
+  border-bottom: 3px solid #f2f7f8;;
+
+  //var(--main-color)
+  margin: 10px 0;
+  display: flex;
+  flex-direction: column;
+  @media (min-width: 667px) and (max-width: 1024px) {
+  }
```

Comment:
I noticed this and some other media queries that were not being used, maybe you forgot to remove them